### ISSUE TITLE
Server marking as read incomming Delta Chat messages

### ISSUE THREAD

**Issue by adbenitez** [mailcow/mailcow-dockerized#4230]
**Prior to placing the issue, please check following:** *(fill out each checkbox with an `X` once done)*
- [x] I understand that not following or deleting the below instructions will result in immediate closure and/or deletion of my issue.
- [x] I have understood that this bug report is dedicated for bugs, and not for support-related inquiries.
- [x] I have understood that answers are voluntary and community-driven, and not commercial support.
- [x] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).

## Summary

Mailcow is marking as seen incoming messages sent with Delta Chat, this is causing that there are no notifications for new messages in the Delta Chat app since the message is marked as read already, my bots in mailcow server stopped working since the messages are marked as read directly they are not passed to the bot as new fresh messages.

TL;DR: the bug was introduced here: 
https://github.com/mailcow/mailcow-dockerized/commit/96a460c2faa97e0776f563ceb81ffdded8a91913#diff-df8548cffb32c4cb35b392d51a17bc99ec34443db95db4b35a26bb9a8c174952R18

## Reproduction

Send a message from from one Delta Chat client to another.

## System information

| Question | Answer |
| --- | --- |
| My operating system | I_DO_REPLY_HERE |
| Is Apparmor, SELinux or similar active? | I_DO_REPLY_HERE |
| Virtualization technlogy (KVM, VMware, Xen, etc - **LXC and OpenVZ are not supported** | I_DO_REPLY_HERE |
| Server/VM specifications (Memory, CPU Cores) | I_DO_REPLY_HERE |
| Docker Version (`docker version`) | I_DO_REPLY_HERE |
| Docker-Compose Version (`docker-compose version`) | I_DO_REPLY_HERE |
| Reverse proxy (custom solution) | I_DO_REPLY_HERE |

- Output of `git diff origin/master`, any other changes to the code? If so, **please post them**.
- All third-party firewalls and custom iptables rules are unsupported. *Please check the Docker docs about how to use Docker with your own ruleset*. Nevertheless, iptabels output can help us to help you: `iptables -L -vn`, `ip6tables -L -vn`, `iptables -L -vn -t nat` and `ip6tables -L -vn -t nat`.
- DNS problems? Please run `docker exec -it $(docker ps -qf name=acme-mailcow) dig +short stackoverflow.com @172.22.1.254` (set the IP accordingly, if you changed the internal mailcow network) and post the output.
